front-end/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+
+FROM node:latest as build-step
```

Comment:
You are using `node:18` on the frontend side, but `node:latest` here.
1. Because both apps use node, you could use the same version for the build images
2. Latest is not often the best solution also. I'd suggest picking the highest available version at the time you are creating the images, but don't use the `latest`. Latest may be good for a long time, maybe forever, but it may lead to breaking changes in the future. _In short: pick concrete image version that is currently working with your app._